### ISSUE TITLE
Loomio

### ISSUE THREAD

**Issue by mikeumus** [bevry-archive/meta#8]
Hey Everyone,

We're trying out @loomio to help us visualize our decentralized decission making process with the goal of streamlining the proposal process.

Our loomio community is at: https://www.loomio.org/g/Jk3fRyU9/bevry

Our first loomio discussion is at: https://www.loomio.org/d/qnDXSrG5?proposal=10367

This issue will serve for discussion about using loomio.

**Comment by balupton** [comment]
Seems nifty.

I hate how it doesn't let you edit your comments, and how there is a character limit on your positions (but not on comments). They should let you edit everything, as typos and kurtness are also evident in the first publish, and they should let you post long position statements with a "read more" link if size is an issue.

Will be interesting to see how it pans out. As currently, a concern is that it could cause a lot of fragmentation from disagreements, as it only uses text which is highly depersonalised and emotionless for disagreeing or agreeing with each other. The catchups have worked really well, as we can work with people's feedback on the fly, react to their emotions politely, and do it all respectfully. Which is a lot harder to do using text.

**Comment by mikeumus** [comment]
Hey @balupton, what I already did was delete comments that weren't on-topic. This can be used in this version of @loomio as a editing feature as well where you delete any comments and repost and the edited version. You voted "block" on the blog decision but I think block means the thing being decided upon is irrelevant or "the powers that be" are blocking voting. Does it let you submit a alternative proposal for Tumblr? Otherwise the thing to do here is vote "no" to using a DocPad powered blog. 

Also important to note that loomio has dedicated themselves as a team to making a effective decission making tool. I'd hold their opinion of what features are present as an expert opinion on the subject (sans the inability to edit comments) the same way @balupton you are the expert opinion on DocPad as it's chief architect. Point being that discussion about the points of why to vote for Tumblr is meant to be discussed outside of loomio in proper detail and that's why I linked to the place where we already discussed it in the Catchup 2. I just realized the reason we're keeping the issue open here in GitHub about the blog is to do just that, discuss in detail. I'll add the link to the github issue in the loomio's "context". Loomio is here to help us come to a quick clear decission.
